ledger/participant-integration-api/src/test/suite/scala/platform/store/appendonlydao/SequentialWriteDaoSpec.scala

```diff
@@ -251,13 +273,35 @@ object SequentialWriteDaoSpec {
     ),
   )
 
-  val updateToDbDtoFixture: Offset => state.Update => Iterator[DbDto] =
+  private val updateToDbDtoFixture: Offset => state.Update => Iterator[DbDto] =
     _ => {
       case r: state.Update.PublicPackageUploadRejected =>
         someUpdateToDbDtoFixture(r.rejectionReason).iterator
       case _ => throw new Exception
     }
 
+  private val dbDtoToStringsForInterningFixture: Iterable[DbDto] => DomainStringIterators = {
+    case iterable if iterable.size == 3 =>
+      new DomainStringIterators(Iterator.empty, List("1").iterator)
+    case _ => new DomainStringIterators(Iterator.empty, Iterator.empty)
```

Comment:
These seem to be hardcoded results that `store correctly in a happy path case` and `start event_seq_id from 1` expect, respectively. Consider moving the fixture definition to each individual test case.